### DISCUSSION TITLE
SBFT Deadline Extension

### DISCUSSION THREAD

**Discussion by ibuysal** [christianbirchler-org/sdc-testing-competition#35] [General]
Hello,
You had previously extended the deadline for the SBFT track. Would it be possible to extend it a little more :) ?

**Comment by ChristianBirchler** [comment]
Hi @ibuysal, thanks for your request. We will discuss it internally. We will let you know asap.

**Comment by ChristianBirchler** [comment]
Hi @ibuysal, we extended the tool submission deadline for SBFT to **January 3rd 2025**.

As the tool submission is closer to the camera-ready deadline now, I kindly ask all participants to prepare a two-page tool description in time. The tool description does not need to be submitted by January 3rd but all participants should prepare for submission during January so that the competition organizers can review and give feedback in time. More details on how to prepare the two-page tool description will follow soon.

I hope the extension helps you to do a submission for the SBFT call. Let us know if there are any issues with that.

In any case; Happy Coding!

Best, Christian
